src/__tests__/app.test.ts

```diff
@@ -64,6 +64,7 @@ const aBonusAPIBasePath = "/bonus/api/v1";
 const aPagoPABasePath = "/pagopa/api/v1";
 const aMyPortalBasePath = "/myportal/api/v1";
 const aBPDBasePath = "/bpd/api/v1";
+const aFIMSBasePath = "/bpd/api/v1";
```

Comment:
Wrong path. Should be `/fims/api/v1`

Reply:
```suggestion
const aFIMSBasePath = "/fims/api/v1";
```

Reply:
fixed in [9682994](https://github.com/pagopa/io-backend/pull/884/commits/9682994fd6ea22ba33c8fb5e3f6076d673ff38e9)